### PR TITLE
initial content which is a boring comment but this is not real work s…

### LINKED ISSUE #10: Als recommendations for things to do/eat
1. Walk the Mars Yard
2. Visit the Von Karman museum

### PR DESCRIPTION
**Pull request by al-niessner**
WIP

Closes #10

### CODE REVIEW

File: al-recommends.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Markup file for demonstrating git development with branches
+
+## Recommendations
+
+### Places to eat
```
Comment (maddj) [inline al-recommends.md:5]: How about some food trucks?

File: al-recommends.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+### Things to do
```
Comment (maddj) [inline al-recommends.md:8]: Comment comment comment

**Review by maddj — DISMISSED** [review]
Comment, comment comment.

File: al-recommends.md (hunk 1 above)
Comment (al-niessner) [reply al-recommends.md:5]: Are there any in La Canada? I found them at Jone's Coffee long time ago.

File: al-recommends.md (hunk 2 above)
Comment (al-niessner) [reply al-recommends.md:8]: Yup. Yup. Yup.

File: al-recommends.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Markup file for demonstrating git development with branches
+
+## Recommendations
```
Comment (maddj) [inline al-recommends.md:3]: asdf

File: al-recommends.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+### Places to eat
+Whatever nonsense
```
Comment (maddj) [inline al-recommends.md:6]: more nonsense please

**Review by maddj — CHANGES_REQUESTED** [review]

**Review by maddj — APPROVED** [review]

**Pull request merged**